gno.land/pkg/gnoweb/webclient_html.go

```diff
@@ -185,9 +186,18 @@ func (s *HTMLWebClient) RenderRealm(w io.Writer, pkgPath string, args string) (*
 		return nil, err
 	}
 
+	// Create a new markdown instance with the current path
+	markdown := goldmark.New(
+		append(s.commonOptions,
+			goldmark.WithExtensions(
+				md.NewGnoExtension(s.domain, pkgPath),
+			),
+		)...,
+	)
```

Comment:
nop, we should not have te recreate a markdown instance each time, 
you need to use goldmark context to pass the path to the extension